demo/alloy-showcase-webapp/src/test/java/com/liferay/faces/test/alloy/showcase/datatable/DataTableTesterBase.java

```diff
@@ -238,37 +184,31 @@ protected String getRowSelectionInputXpath(Customer customer) {
 			"')]//input[contains(@id,'customers:')]";
 	}
 
-	/**
-	 * Determines whether or not the specified page number is active in the rendered <code>alloy:paginator</code>
-	 * markup.
-	 */
-	protected ExpectedCondition<Boolean> isPaginatorPageNumberButtonActive(int pageButtonNumber) {
```

Comment:
@ngriffin7a, those commits will break the test. You would need to do that inside the loop as well to ensure that the `apply()` method is called and the value of `navigationButtonDisabled` is updated and on every iteration.